drivers/gles3/shaders/scene.glsl

```diff
@@ -520,10 +514,8 @@ void main() {
 	}
 
 #if defined(NORMAL_USED) || defined(TANGENT_USED) || defined(NORMAL_MAP_USED) || defined(LIGHT_ANISOTROPY_USED)
-
-	vec3 binormal;
+	vec3 tangent, binormal;
```

Comment:
```suggestion
	vec3 tangent;
	vec3 binormal;
```
We prefer one declaration per line for readability, it's fine to leave existing code that breaks this alone but I don't think we should turn code following this rule into code that breaks it

Reply:
Understandable, these have now been restored.